net.voltaicideas.dupeGuru.json

```
@@ -0,0 +1,62 @@
+{
+    "app-id": "net.voltaicideas.dupeGuru",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
```

```suggestion
    "runtime-version": "5.15-22.08",
```

this is the current one